include/bl_crypto.h

```diff
@@ -34,7 +34,7 @@ extern "C" {
 	#define SHA256_CTX_SIZE sizeof(nrf_cc310_bl_hash_context_sha256_t)
 	typedef nrf_cc310_bl_hash_context_sha256_t bl_sha256_ctx_t;
 #else
-	#define SHA256_CTX_SIZE 128
+	#define SHA256_CTX_SIZE 224
```

Comment:
We could bump it to something like 256, to handle any further increases in size as well.

Reply:
K